### PR TITLE
Add support for ejb subsystem configuration.

### PR DESCRIPTION
**Pull request by claudio4j**

### CODE REVIEW

File: app/src/main/java/org/jboss/hal/client/AppModule.java [hunk 1]
```diff
@@ -194,6 +196,11 @@ protected void configure() {
                 IiopPresenter.MyView.class,
                 Mbui_IiopView_Provider.class,
                 IiopPresenter.MyProxy.class);
+        
+        bindTemplatedPresenter(EjbPresenter.class,
```
Comment (hpehl) [inline app/src/main/java/org/jboss/hal/client/AppModule.java]: Try to keep the bind*** methods in order (A-Z)

File: app/src/main/java/org/jboss/hal/client/configuration/subsystem/ejb/EjbPresenter.java [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+/**
+ * @author Claudio Miranda
+ */
+public class EjbPresenter extends MbuiPresenter<EjbPresenter.MyView, EjbPresenter.MyProxy> {
+
+    // @formatter:off
+    @ProxyCodeSplit
+    @NameToken(NameTokens.EJB3)
+    @Requires({EJB_SUBSYSTEM_ADDRESS, THREAD_POOL_ADDRESS, REMOTING_PROFILE_ADDRESS, BEAN_POOL_ADDRESS, CACHE_ADDRESS,
```
Comment (hpehl) [inline app/src/main/java/org/jboss/hal/client/configuration/subsystem/ejb/EjbPresenter.java]: Pretty much resources in here. How's the performance when first loading the EJB application? 

The r-r-d op uses `recursive-depth=2` by default. So a simple `@Requires(EJB_SUBSYSTEM_ADDRESS)` should work?

File: core/src/main/java/org/jboss/hal/core/mbui/form/PropertyFilter.java [hunk 3]
```diff
@@ -38,6 +38,11 @@ public boolean test(final Property property) {
                 (p.getValue().hasDefined(REQUIRED) && p.getValue().get(REQUIRED).asBoolean()) ||
                         (p.getValue().hasDefined(NILLABLE) && !p.getValue().get(NILLABLE).asBoolean()));
 
+        // do not include "deprecated" attribute
```
Comment (hpehl) [inline core/src/main/java/org/jboss/hal/core/mbui/form/PropertyFilter.java]: I think that's ok. Don't know if we ever need to include a deprecated attribute. If so this wouldn't work.

**Comment by claudio4j** [comment]
Fixed the code as requested.

**Pull request merged**